### PR TITLE
feat(udf): add `body` field for udf body definition

### PR DESCRIPTION
**Pull request by xzhseh**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?
As titled.
This branch is based on #14139, just for review convenience.

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added test labels as necessary. See [details](https://github.com/risingwavelabs/risingwave/blob/main/docs/developer-guide.md#ci-labels-guide).
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [x] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: src/meta/model_v2/src/function.rs [hunk 1]
```diff
@@ -41,6 +41,7 @@ pub struct Model {
     pub language: String,
     pub link: String,
     pub identifier: String,
+    pub body: String,
```
Comment (wangrunji0408) [inline src/meta/model_v2/src/function.rs:44]: We can make it an Option.

Comment (yezizp2012) [inline src/meta/model_v2/src/function.rs:44]: Also need to add column `body` in two places in this file: https://github.com/risingwavelabs/risingwave/blob/7c84d06ff4f8b705117b862619a1f6766ca2d4c0/src/meta/model_v2/migration/src/m20230908_072257_init.rs#L1101
https://github.com/risingwavelabs/risingwave/blob/7c84d06ff4f8b705117b862619a1f6766ca2d4c0/src/meta/model_v2/migration/src/m20230908_072257_init.rs#L710

Don't forget to remove `not_null` constraint if body is optional.

**Review by yezizp2012 — COMMENTED** [review]
Rest LGTM.

File: src/meta/model_v2/src/function.rs (hunk 1 above)
Comment (xzhseh) [reply src/meta/model_v2/src/function.rs:44]: Does `Model` need the value of `body`?
If not I'll just insert `None` whenever this field is needed.

Comment (yezizp2012) [reply src/meta/model_v2/src/function.rs:44]: Yes, it's necessary.

**Comment by xzhseh** [comment]
If it looks good now I'll merge it, cc @yezizp2012 @wangrunji0408.

**Review by yezizp2012 — COMMENTED** [review]
May also change it as optional in proto and function catalog definition. Rest LGTM.

**Review by yezizp2012 — APPROVED** [review]

**Review by wangrunji0408 — APPROVED** [review]

**Pull request merged**